itez/beneficiary/models.py

```diff
@@ -34,10 +34,49 @@ class District(models.Model):
 
     province = models.ForeignKey(
         Province,
-        on_delete=PROTECT,
+        on_delete=PROTECT
     )
 
     created = models.DateTimeField(auto_now_add=True)
 
     def __str__(self):
         return self.name
+
+
+class WorkDetail(models.Model):
+    """
+    Include Work Detail properties.
+    """
+    POSITIVE_CHOICE = "Yes"
+    NEGATIVE_CHOICE = "No"
+    CHOICE_LIST = [
+        (POSITIVE_CHOICE, "Yes"),
+        (NEGATIVE_CHOICE, "No")
```

Comment:
the names are defaults just as python would name each boolean value

i dont think we need this for a simple `yes` or `no` attrbute